client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -34,7 +36,7 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 				<div className="go-mobile__row">
 					<img
 						className="go-mobile__icon"
-						src={ wpToJpImage }
+						src={ isRtl ? wpToJpImageRtl : wpToJpImage }
```

Comment:
I wonder if instead of creating another svg, we could just add some CSS to mirror the image by the X-axis:

```
transform: scaleX(-1);
```

We could do that either directly with inline style directly applied to the image, or even better - with a classname to the `img`, and specify that style in a scss file.

What do you think?

Reply:
@tyxla, thank you for taking a look at this! I like the idea of using CSS to avoid an additional file. However, although the positioning of the WP and JP logos are 'flipped' for RTL, the individual logos themselves need to stay the same. Using CSS causes everything to flip, leading to an odd look for the individual logos:

| Actual RTL logo  | CSS transform |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/204024297-bd7f12fb-47ab-4bec-a250-6e9cd468a45f.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/204024171-97f4ce6e-ddd3-486f-8398-824c21d642e5.png" width="100%"> |

I'm not sure there's a way around the additional file, but let me know if I'm missing something else here!

Reply:
We could potentially make the svg a React component, and add a `isRtl` prop to it. Then, when it `isRtl` is true, we'd add `transform="scaleX(-1)"` to each of the `path` elements.

That makes things more complicated, though, so your solution with an extra SVG sounds good 👍

Reply:
I'll keep in mind that option for the future, in case we end up having a lot of RTL variations. Thanks!